CODEOWNERS

```diff
@@ -2,10 +2,10 @@
 # Last matching pattern takes preecedence.
 
 # Default owners for everything in repo.
-*  @lamberta @MarkDaoust @yashk2810 @anirudh161
+*  @lamberta @MarkDaoust @8bitmp3
```

Comment:
Did you rebase? This update is already checking in master: https://github.com/tensorflow/docs/blob/master/CODEOWNERS

Reply:
I rebased this PR with master and force pushed to fix this.